README.adoc

```diff
@@ -154,6 +154,40 @@ export EMQTTB_METRICS__GRAFANA__URL="http://localhost:3000"
 emqttb --grafana ...
 ----
 
+== Build and run locally
+
+Requirements:
+
+* https://www.erlang.org/[Erlang/OTP 24+]
+* https://rebar3.org/[rebar3]
+* make
+* cmake (for quicer)
+* tools for building docs
+** asciidoctor (lee dependency)
+** xsltproc
+** docbook-xsl
+** java (jre is enough)
```

Comment:
```suggestion
If you want to build HTML documentation and manpages, it's necessary to install the following tools:
** asciidoctor (lee dependency)
** xsltproc
** docbook-xsl
** java (jre is enough)

This can be omitted by running `export CAN_BUILD_DOCS=false`.
```